projects/packages/search/changelog/update-search-record-meter-notice-box-fix

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+updated notice box content when site is not indexed
```

Comment:
Not sure if we still need to include the 'Search: ' prefix on changelog entries now we're committing to the plugin? I have asked @anomiex about this on P2: p9dueE-2b7-p2#comment-8110.